pages/treeherder.py

```diff
@@ -13,9 +14,16 @@
 
 class TreeherderPage(Base):
 
+    _clear_all_menu_locator = (By.CSS_SELECTOR, 'ul.dropdown-menu:nth-child(3) > li:nth-child(4) > a:nth-child(1)')
     _job_details_actionbar_locator = (By.ID, 'job-details-actionbar')
     _job_result_status_locator = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span')
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
+    _logviewer_pin_job_locator = (By.CSS_SELECTOR, 'a > span.glyphicon.glyphicon-pushpin')
+    _next_job_title = (By.CSS_SELECTOR, '[class*="btn job-btn btn-dkgray"]')
+    _open_save_menu_locator = (By.CSS_SELECTOR, 'span.dropdown-toggle')
+    _pinboard_item_locator = (By.CSS_SELECTOR, 'span.btn-group')
+    _pinboard_locator = (By.ID, 'pinboard-panel')
+    _pinboard_remove_job_locator = (By.CSS_SELECTOR, 'i.fa-times')
```

Comment:
Although this will return the first pinned job's remove element, I think `#pinned-job-list .pinned-job-close-btn` would be more suitable.